frontend/src/campaign/speech.js

```diff
@@ -192,10 +228,10 @@ export class Speech extends React.Component {
                     {topic}
                 </div>
                 <div className='speech-option_btns'>
-                    {[...Array(5).keys()].map((score, j) => (
+                    {[...Array(3).keys()].map((score, j) => (
```

Comment:
I should've caught this in an earlier PR, but I'm noticing it now, so going to ask for change -- this kind of pattern is pretty hard to read, when it's doing something very simple (a map over [0,1,2]). My preference would be to make a React fragment inside of a normal for loop earlier in the render call.

`{[...Array(3).keys()].map((score, j) => (` technically works but it's pretty inscrutable! Plus `score` and `j` are totally redundant. Even `[0,1,2].map((score) =>` is clearer.